frontend/src/metabase/parameters/components/FilterApplyButton/FilterApplyButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback, useRef } from "react";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+
+import {
+  getHasUnappliedParameterValues,
+  getIsAutoApplyFilters,
+} from "metabase/dashboard/selectors";
+import { applyDraftParameterValues } from "metabase/dashboard/actions";
+import { ApplyButton } from "./FilterApplyButton.styled";
+
+export default function FilterApplyButton() {
+  const isAutoApplyFilters = useSelector(getIsAutoApplyFilters);
+  const hasUnappliedParameterValues = useSelector(
+    getHasUnappliedParameterValues,
+  );
+
+  const dispatch = useDispatch();
+  const handleApplyFilters = useCallback(() => {
+    dispatch(applyDraftParameterValues());
+  }, [dispatch]);
+
+  const applyButtonRef = useRef(null);
```

Comment:
nit: seems not used

Reply:
Nice catch. It was from when I was using `react-transition-group`. I'll need to see why ESLint doesn't catch this.

Reply:
Ok, I see that it's referenced, but it's not really used for anything that's why ESLint didn't catch it.

Reply:
Fixed